docs/geedocs/docsrc/answer/portable/portableGlobeServer5_2.rst

```diff
@@ -195,34 +170,31 @@ Portable Globe and Server 5.2
          Use caution when changing them as they may dramatically
          increase build times and globe sizes.
 
-      .. _Globe_Name:
-      .. rubric:: Globe name
+      .. _Portable_Name:
+      .. rubric:: Portable Name
 
-      The globe name defines the file name of this globe; it will be
-      created as a ``.glb`` file, which is a single-file format for
-      sharing Google Earth Enterprise globes. Spaces, slashes, and
-      double dots (..) will be converted to underscores in the saved
-      globe name.
+      The name defines the file name of the portable - a single-file format for
+      sharing GEE maps and globes. Spaces, slashes, and double dots (..) will be
+      converted to underscores in the final portable filename.
 
-      We recommend using a descriptive name for each globe, so that
-      others will know what geographic area, or what mission, the globe
+      It's recommend to use a descriptive name for each portable, so that
+      others will know the area of interest or the purpose the portable
       was created for.
 
       .. note::
 
-         Building a globe will overwrite any existing globes
-         with the same name. If multiple users are cutting globes, we
+         Building a portable will overwrite any existing portables
+         with the same name. If multiple users are Cutter, it's
          recommend assigning unique prefixes to each user for their
-         globe names to ensure that files are not accidentally
+         portable names to ensure that files are not accidentally
          overwritten.
 
       .. _Drawing_Polygon:
-      .. rubric:: Drawing the polygon
+      .. rubric:: Drawing the Polygon
 
-      Once the globe name has been specified, you can define the
-      geographic region to be cut by drawing a polygon on the globe.
-      There are two ways to draw the polygon, one of which has been
-      deprecated.
+      Once the portable name has been specified, you can define the
+      geographic region to be cut by drawing a polygon. There are two ways to
+      draw the polygon, one of which has been deprecated.
 

```

Comment:
If this is the same as the Spartan viewer Cutter updates, pull in updates from this PR
https://github.com/google/earthenterprise/pull/1614